api/cpp/cmake/SlintMacro.cmake

```diff
@@ -1,22 +1,37 @@
 # Copyright © SixtyFPS GmbH <info@slint-ui.com>
 # SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
 
+# Set up machinery to handle SLINT_EMBED_RESOURCES target property
+set(DEFAULT_SLINT_EMBED_RESOURCES OFF CACHE BOOL
+    "The default resource embedding option to pass to the slint compiler")
```

Comment:
```suggestion
    "The default resource embedding option to pass to the Slint compiler")
```

Reply:
As documentation I think just this sentence is insufficient. I suggest to add a section to `cmake.md`, where we already document aspects like backend selection, and explain there how this new feature can be used.